File: week-01/day1/99Bottles.js
function bottles(num) {
  if (num <=0) {
    console.log("No more bottles of beer on the wall, no more bottles of beer. \n Go to the store and buy some more, 99 bottles of beer on the wall.")
  }
  if (num == 2) {
    console.log(`${num} bottles of beer on the wall, ${num} bottles of beer. \n Take one down and pass it around, ${num -1} bottle of beer on the wall.`)
    return bottles(num-1)}
  while (num >= 3) {
    console.log(`${num} bottles of beer on the wall, ${num} bottles of beer. \n Take one down and pass it around, ${num -1} bottles of beer on the wall.`)
    return bottles(num - 1)
  }
}
bottles(99)

File: week-01/day3/99bottles.py
def ninety_bottles(num):
    starting_num = num
    while num > 0:
        if num == 1:
            print(f"One bottle of beer on the wall, one bottle of beer. \nTake one down, pass it around, no bottles of beer on the wall. \nNo more bottles of beer on the wall, no more bottles of beer. \nGo to the store and buy some more, {starting_num} bottles of beer on the wall.")
        elif num == 2: 
            print("Two bottles of beer on the wall, two bottles of beer. \nTake one down and pass it around, one bottle of beer on the wall.")
        elif num > 2:
            print(f"{num} bottles of beer on the wall, {num} bottles of beer. \nTake one down and pass it around, {num} bottles of beer on the wall.")
        num -= 1
print(ninety_bottles(1))
